stories/feed-input/feed-input.stories.js

```diff
@@ -0,0 +1,90 @@
+        <button class="fd-button sap-icon--feeder-arrow fd-feed-input__submit-button" aria-label="Submit" title="Submit" aria-disabled="true" disabled></button>
+    </div>
+`;
+
+empty.storyName = 'Empty';
+empty.parameters = {
+    docs: {
+        storyDescription: 'Example with an empty text field'
+    }
+};
+
+
+export const notEmpty = () => `
+    <div class="fd-feed-input" role="region" aria-label="Feed input with one row text">
+        <div class="fd-feed-input__thumb" style="background-image: url(http://lorempixel.com/460/620/people/7/)" role="img" title="Thumbnail"></div>
+        <textarea class="fd-textarea fd-feed-input__textarea" id="textarea-2" aria-label="Feed" rows="1" placeholder="Post something here">Textarea with a text makes submit button active</textarea>
+        <button class="fd-button sap-icon--feeder-arrow fd-feed-input__submit-button" aria-label="Submit" title="Submit" aria-disabled="false"></button>
+    </div>
+`;
+
+notEmpty.storyName = 'With some text';
+notEmpty.parameters = {
+    docs: {
+        storyDescription: 'Example with a one row text'
+    }
+};
+
+export const inputGrowth = () => `
+    <div class="fd-feed-input" role="region" aria-label="Feed input height growth">
+        <div class="fd-feed-input__thumb" style="background-image: url(http://lorempixel.com/460/620/people/7/)" role="img" title="Thumbnail"></div>
+        <textarea class="fd-textarea fd-feed-input__textarea" id="textarea-3" aria-label="Feed" rows="3" placeholder="Post something here">With every line\ntextarea growths up\nuntil it reaches the max height</textarea>
+        <button class="fd-button sap-icon--feeder-arrow fd-feed-input__submit-button" aria-label="Submit" title="Submit" aria-disabled="false"></button>
+    </div>
+`;
+
+inputGrowth.storyName = 'Multiple lines';
+inputGrowth.parameters = {
+    docs: {
+        storyDescription: 'Every new line extends textarea height until maximum height is reached'
+    }
+};
+
+export const overMaxHeight = () => `
+    <div class="fd-feed-input" role="region" aria-label="Feed input over max height">
+        <div class="fd-feed-input__thumb" style="background-image: url(http://lorempixel.com/460/620/people/7/)" role="img" title="Thumbnail"></div>
+        <textarea class="fd-textarea fd-feed-input__textarea" id="textarea-4" rows="18" aria-label="Feed" placeholder="Post something here">Over max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height</textarea>
+        <button class="fd-button sap-icon--feeder-arrow fd-feed-input__submit-button" aria-label="Submit" title="Submit" aria-disabled="false"></button>
+    </div>
+`;
+
+overMaxHeight.storyName = 'Over max height';
```

Comment:
Is it possible to name this "Max height" or "Maximum"?

And add this as a description: "**The maximum feed input displays a text field with the maximum amount of text lines it can contain. Once the text area has reached the maximum height, the text area becomes scrollable.**"

*******Question about the last part though — is the text area really only scrollable ONCE the max is reached? Or can it be scrolled before then?**

Reply:
Answer: **When the maximum height is achieved, a scrollbar is displayed inside the Input Field, to access the overflowing text**

Reply:
Very good! Just to not confuse readers between "feed input and input field," I think this would work nicely: 

"**The maximum feed input displays a text field with the maximum amount of text lines it can contain. Once the text field has reached the maximum height, a scrollbar appears to access the overflowing text.**"